libs/components/datetime/src/lib/modules/datepicker/datepicker.service.ts

```diff
@@ -2,9 +2,11 @@ import { Injectable } from '@angular/core';
 
 import { Subject } from 'rxjs';
 
-import { SkyDatepickerDate } from './datepicker-date';
+import { SkyDatepickerDate } from './calendar/datepicker-date';
 
 /**
+ * Manages the state of the calendar picker.
+ * TODO: Consider changing the name of this service to better fit its purpose.
  * @internal
```

Comment:
same question as `SkyDatepickerDate`: why not make that change now since these are internal; there are no references to either of these in ADO

Reply:
Done!